ci-operator/step-registry/ovn/conf/ipsec-manifest/ovn-conf-ipsec-manifest-commands.sh

```diff
@@ -7,19 +7,9 @@ cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
 apiVersion: operator.openshift.io/v1
```

Comment:
Can you change the commit message title to something like:
"ipsec: simplify config"

You can leave out "I'm using this in the refreshed installation docs and as near as I can tell, this works and is valid."